.github/workflows/tx-pull.yml

```diff
@@ -0,0 +1,78 @@
+name: Transifex sync
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  tx-sync:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Transifex client
+      run: |
+        curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+        chmod +x ./tx
+    - name: Pull translations from Transifex
+      run: |
+        ./tx -t ${{ secrets.TX_TOKEN }} pull -a -f
+    - uses: stefanzweifel/git-auto-commit-action@49620cd3ed21ee620a48530e81dba0d139c9cb80
+      with:
+        # Optional. Commit message for the created commit.
+        # Defaults to "Apply automatic changes"
+        commit_message: "chore: Pull transifex translations"
+
+        # Optional. Local and remote branch name where commit is going to be pushed
+        #  to. Defaults to the current branch.
+        #  You might need to set `create_branch: true` if the branch does not exist.
+        branch: i18n-sync
+
+        # Optional. Options used by `git-commit`.
+        # See https://git-scm.com/docs/git-commit#_options
+        commit_options: '--no-verify --signoff'
+
+        # Optional glob pattern of files which should be added to the commit
+        # Defaults to all (.)
+        # See the `pathspec`-documentation for git
+        # - https://git-scm.com/docs/git-add#Documentation/git-add.txt-ltpathspecgt82308203
+        # - https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec
+        file_pattern: public/locales
+
+        # Optional. Local file path to the repository.
+        # Defaults to the root of the repository.
+        repository: .
+
+        # Optional. Options used by `git-add`.
+        # See https://git-scm.com/docs/git-add#_options
+        add_options: '-A'
+
+        # Optional. Options used by `git-push`.
+        # See https://git-scm.com/docs/git-push#_options
+        push_options: '--force'
+
+        # Optional. Disable dirty check and always try to create a commit and push
+        skip_dirty_check: true
+
+        # Optional. Skip internal call to `git fetch`
+        skip_fetch: true
+
+        # Optional. Skip internal call to `git checkout`
+        skip_checkout: true
+
+        # Optional. Prevents the shell from expanding filenames.
+        # Details: https://www.gnu.org/software/bash/manual/html_node/Filename-Expansion.html
+        disable_globbing: true
+
+        # Optional. Create given branch name in local and remote repository.
+        create_branch: true
+    - name: pull-request
+      uses: repo-sync/pull-request@65785d95a5a466e46a9d0708933a3bd51bbf9dde
+      with:
+        source_branch: "i18n-sync"
+        destination_branch: "main"
```

Comment:
@SgtPooki fyi we missed the fact this repo uses `master` – fixed in https://github.com/ipfs/ipld-explorer-components/commit/006c50d5db48ff243c8b443eab8fc7be0013885b